core/workspace.js

```diff
@@ -55,6 +56,16 @@ Blockly.Workspace = function(opt_options) {
    * @private
    */
   this.topBlocks_ = [];
+  /**
+   * @type {!Array.<!Blockly.WorkspaceComment>}
+   * @private
+   */
+  this.topComments_ = [];
```

Comment:
Is top comments really just all workspace comments? I assume that comments can't be nested/connected the way blocks can..

Reply:
It's currently all workspace comments, yes.  Block comments *are* nested in a similar way to blocks.  If we find a way to merge block and workspace comments, this may be important.